docs/desktop/gnome/file-shredder.md

```diff
@@ -1,53 +1,67 @@
 ---
-title: File Shredder 
-author: Christine Belzie
-contributors: Steven Spencer, Ganna Zhyrnova 
+title: File Shredder - Secure Deletion
+author: Wale Soyinka
+contributors:
+tags:
+  - gnome
+  - desktop
+  - security
+  - privacy
+  - flatpak
 ---
- 
-## Introduction
 
-Do you need to remove a postcard or a PDF file containing information like your birthday? Check out File Shredder, an application that permanently removes sensitive information online.
+# **File Shredder: Permanently and Securely Delete Files**
 
```

Comment:
Same here "File Shredder comes down from the meta as the level 1 heading and then just replace the Introduction with ## **Permanently and securely delete files**